client/my-sites/stats/stats-purchase/stats-purchase-notice.jsx

```diff
@@ -3,6 +3,7 @@ import config from '@automattic/calypso-config';
 import page from '@automattic/calypso-router';
 import { Button } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
+import { STATS_PRODUCT_NAME } from 'my-sites/stats/constants';
```

Comment:
This is causing a build error. Not sure why it worked in my local build...

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/99485/commits/32ac085ef81f14d9f5bd00340568b140f0736b58.